plugins/rias/ls.rias.js

```diff
@@ -83,9 +83,20 @@
 		};
 
 		setOption = function(attr, run){
-			var attrVal = elem.getAttribute('data-'+ attr) || elemStyles.getPropertyValue('--ls-' + attr).trim() || null;
```

Comment:
You can make this a lot shorter: `(elemStyles.getPropertyValue('--ls-' + attr) || '').trim()`